proposals/0001-simd-process.md

```diff
@@ -0,0 +1,141 @@
+
+## Rationale
+
+A proposal is intended to be reviewed by core engineering and community members
+keeping in mind security concerns, tradeoffs, and backwards compatibility.
+Having a proposal process helps identify design issues early, alert the
+community on a change, helps scale newer contributors on the architecture, and
+acts as a historical record of the design decisions that have gone into Solana.
+For implementors, the proposal is a blueprint for the feature and helps track
+the development of a feature.
+
+## When you need to follow this process
+
+You need to follow this process if you intend to make "substantial" changes to
+the Validator, RPC, consensus, or a change to the proposal process itself. What
+constitutes a "substantial" change is evolving based on community norms and
+varies depending on what part of the ecosystem you are proposing to change, but
+may include the following:
+
+- A change in format of a RPC API method
+- Networking interface changes between validators
+- Compute requirement changes on the runtime
+
+Some changes do not require a proposal:
+
+- Rephrasing, reorganizing, refactoring, or otherwise "changing shape does not
+change meaning".
+- Additions that strictly improve objective, numerical quality criteria
+(warning removal, speedup, better platform coverage, more parallelism, trap
+more errors, etc.)
+
+## Proposal Types
+
+There are two types of proposals:
+
+- A **Standard** Proposal describes any change that affects most or all Solana
+implementations, such as a change to the network protocol, consensus, proposed
+application standards/conventions, block or transaction validity, or any change
+or addition that affects the interoperability of applications using Solana.
+Standard proposals can be broken down into the following categories:
+
+  - **Core**: Anything that affects consensus or substantial changes to the
+validator.
+  - **Networking**: Changes or substantial improvements to network protocol
+specifications.
+  - **Interfaces**: Improvements around the client JSON RPC API specifications
+and standards. 
+
+- A **Meta** Proposal describes a process surrounding Solana or proposes a
+change to (or an event in) a process. Process Proposals are like Standard
+Proposals but apply to areas other than the Solana protocol itself. They may
+propose an implementation, but not to Solana's codebase; they often require
+community consensus; unlike Informational Proposals, they are more than
+recommendations, and users are typically not free to ignore them. Examples
+include procedures, guidelines, changes to the decision-making process, and
+changes to the tools or environment used in Solana development. Any meta-EIP is
+also considered a Process Proposals.
+
+## Proposal Lifecycle
+
+The stages in a lifecycle of a proposal are as follows:
+
+- Idea
+- Draft
+- Review
+- Accepted
+- Stagnant
+- Withdrawn
+- Implemented
+
+![Proposal Process](../resources/proposal-process.png "Proposal Process")
```

Comment:
How about a Mermaid Markdown diagram instead of png?

```mermaid
flowchart LR
  Idea
  Draft
  Review
  Accepted

  subgraph fail[&nbsp];
    Stagnant
    Withdrawn
  end

  style fail fill:#ffe8e7,stroke:none
  style Accepted fill:#f3f8dc,stroke:#c3db50;

  Idea ---> Draft;
  Draft ---> Review;
  Review ---> Accepted;

  Draft ---> Stagnant;
  Review ---> Stagnant;
  Review ---> Withdrawn;
```

Reply:
TIL. This is a great addition to markdown on github. Updated